tests/api/organisations/test_organisations_rest_api.py

```diff
@@ -86,11 +86,14 @@ def test_organisation_secure_api_update(client, json_header, org_martigny,
     assert res.status_code == 403
 
 
-def test_location_can_delete(client, org_martigny, lib_martigny):
+def test_organisation_can_delete(client, org_martigny, lib_martigny,
+                                 acq_receipt_fiction_martigny):
     """Test can delete an organisation."""
     can, reasons = org_martigny.can_delete
     assert not can
+    assert 'links' in reasons
```

Comment:
Maybe you can delete this, because `assert reasons['links']['libraries']` will fail if `links` is missing.